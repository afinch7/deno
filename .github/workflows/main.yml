name: Rust

on: [push, pull_request]

jobs:
  build:

    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v1
    - name: Check Format
      run: ./tools/test_format.py
    - name: Lint
      run: ./tools/lint.py
