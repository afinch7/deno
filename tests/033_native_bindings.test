args: run --reload --allow-dlopen tests/033_native_bindings.ts
output: tests/033_native_bindings.ts.out