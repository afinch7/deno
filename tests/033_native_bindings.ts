import { testOp } from "../test_binding/main.ts";

console.log(testOp({ data: "test", zeroCopyData: { some: "data" } }));
