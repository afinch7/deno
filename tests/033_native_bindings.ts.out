Hello from native bindings. data: "test" | zero_copy: {"some":"data"}
test
